### DISCUSSION TITLE
Is there a problem about github merge branch endpoint?

### DISCUSSION THREAD

**Discussion by nedimYilmaz** [community/community#40842] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I am trying to merge one branch to another with `https://api.github.com/repos/OWNER/REPO/merges` endpoint but its giving an error like:

```
{
  "message": "Head does not exist",
  "documentation_url": "https://docs.github.com/rest/branches/branches#merge-a-branch"
}
```

I checked the repository branches, commit's shas'. All of them valid but the endpoint is returning an error to me.
Related documentation: https://docs.github.com/en/rest/branches/branches?apiVersion=2022-11-28#merge-a-branch

The problem started today.

**I tried another github repository and it works. So i will investigate the problem. You can ignore the this discussion.**

**Comment by ossanna16** [comment]
Hi there @nedimYilmaz and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
